src/components/LimitedWidthContainer.tsx

```diff
@@ -1,13 +1,18 @@
 import { css } from '@emotion/react'
 import styled from '@emotion/styled'
 
-export const limitedWidthContainerStyle = css`
-  --max-inner-width: calc(1440px - var(--sidenav-collapsed-width) - calc(2 * var(--global-horizontal-padding)));
+type LimitedWidthContainerProps = { big?: boolean }
+
+export const limitedWidthContainerStyle = (props: LimitedWidthContainerProps) => css`
+  --max-inner-width: calc(
+    ${props.big ? '2284' : '1440'}px - var(--sidenav-collapsed-width) - calc(2 * var(--global-horizontal-padding))
+  );
```

Comment:
This value isn't correct, it's `1304px`, should be `1376px`, we basically shouldn't subtract sidenav width